packages/lab/src/date-input/DateInput.tsx

```diff
@@ -239,8 +239,10 @@ export const DateInput = forwardRef<
   // Update date strings when dates change
   useEffect(() => {
```

Comment:
This only occurs to me now, should any of the string value update in the same render cycle?